bls-signatures-ffi/src/lib.rs

```diff
@@ -83,71 +97,69 @@ pub unsafe extern "C" fn aggregate(
 }
 
 /// Verify that a signature is the aggregated signature of hashes - pubkeys
-/// WARNING: This function can panic if there are a different number of digests and keys
 ///
 /// # Arguments
 ///
 /// * `signature_ptr`             - pointer to a signature byte array (SIGNATURE_BYTES long)
 /// * `flattened_digests_ptr`     - pointer to a byte array containing digests
 /// * `flattened_digests_len`     - length of the byte array (multiple of DIGEST_BYTES)
 /// * `flattened_public_keys_ptr` - pointer to a byte array containing public keys
-/// * `flattened_public_keys_len` - length of the byte array (multiple of PUBLIC_KEY_BYTES)
```

Comment:
Note: You dropped the comment, but not the corresponding parameter.

Reply:
ups